build-tools/gradle/gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```

Comment:
Does the commit message say why we need this? It seems like it would set it _lower_ to me.

Reply:
That, again, comes from gradle upgrade - I did nothing here except to run the upgrade procedure